Sources/Kitura/Router.swift

```diff
@@ -726,8 +733,11 @@ extension Router : HttpServerDelegate {
             sendResourceIfExisting(routeResp, resource: "index.html")
         } else {
             do {
-                try routeResp.status(HttpStatusCode.NOT_FOUND).send("Cannot \(String(routeReq.method).uppercased()) \(routeReq.parsedUrl.path ?? "").").end()
-            } catch {}
+                let message = "Cannot \(String(routeReq.method).uppercased()) \(routeReq.parsedUrl.path ?? "")."
+                try routeResp.status(HttpStatusCode.NOT_FOUND).send(message).end()
```

Comment:
This will conflict against https://github.com/IBM-Swift/Kitura-net/pull/22 so depending on merge order we will need to fix up.